build-tools/api-xml-adjuster/README.md

```diff
@@ -0,0 +1,32 @@
+# Mono.Android API XML generator
```

Comment:
I think this file should instead be in `$(topdir)/Documentation/BindingApiLevels.md`.